From 92dadbe05a5b0615149e143a723193d2e7b866fb Mon Sep 17 00:00:00 2001
From: Hariharan K <quic_harihk@quicinc.com>
Date: Sat, 8 Jul 2023 12:33:34 +0530
Subject: [PATCH] scripts: Makefile: Add __CPU_THERMAL__ to dtc flags

Adding __CPU_THERMAL__ to dtc_cpp_flags to select
thermal/thermald dtsi.

Change-Id: I2792853c53213789fb962db2c994ad98f115dbce
Signed-off-by: Hariharan K <quic_harihk@quicinc.com>
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index ccc2b68aaf7d..dc83d1f0deac 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -248,6 +248,10 @@ dtc_cpp_flags  = -Wp,-MMD,$(depfile).pre.tmp -nostdinc                    \
 		 $(addprefix -I,$(DTC_INCLUDE))                          \
 		 -undef -D__DTS__
 
+ifeq ($(CONFIG_CPU_THERMAL),y)
+dtc_cpp_flags += -D __CPU_THERMAL__
+endif
+
 ifdef CONFIG_OBJTOOL
 
 objtool := $(objtree)/tools/objtool/objtool
-- 
2.34.1

